pmm-app/src/pmm-check/Check.service.ts

```diff
@@ -56,13 +56,20 @@ export const CheckService = {
 export const processData = (data: Alert[]): ActiveCheck[] => {
   const result: Record<
     string,
-    Array<{ summary: string; description: string; severity: string; labels: { [key: string]: string } }>
+    Array<{
+      summary: string;
+      description: string;
+      severity: string;
+      labels: { [key: string]: string },
+      silenced: boolean;
+     }>
```

Comment:
Is it worth it typing this?

Reply:
That was my though as well, but it was there and I don't want to refactor that too much now : ) I just added what was missing, but I think we should consider refactoring that code in the future